fixes.inc

```diff
@@ -9796,6 +9844,31 @@ native BAD_SetPlayerName(playerid, const name[]) = SetPlayerName;
 		// =====================
 		//  BEGIN: OnVehicleMod
 		// =====================
+
+		switch(componentid)
+		{
```

Comment:
You switch on `componentid`, then once you've determined the exact value, use `componentid` again.  All this switch really does is determine which of the five variables you want to store the component ID in, but the values are always constant.  So, again, use one variable for everything (in a bit map).

Reply:
Actually, shouldn't this be switching on component type, not on component ID?  In which case, that does drastically alter most of my comments because it is no long a simple yes/no to the components existing.   That might take a little bit more finagling to get them in one variable and might not be worth it in that case.

Reply:
Yes. It should be component type. I pushed 5 mins ago.